.github/workflows/release.yml

```
@@ -55,7 +55,7 @@ jobs:
         labels: ${{ steps.docker-meta.outputs.labels }}
 
     - name: Attest image
-      uses: github-early-access/generate-build-provenance@main
+      uses: actions/attest-build-provenance@951c0c5f8e375ad4efad33405ab77f7ded2358e4  # v1.1.1
```

Why not use directly the tag?

```suggestion
      uses: actions/attest-build-provenance@v1.1.1
```

---

tags are not immutable. When possible we try to use the sha commits, as explained in https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions

> Pin actions to a full length commit SHA


Dependabot works also in this particular case and besides of bumping the hash it will also change the comment, see https://github.com/elastic/apm-pipeline-library/pull/2604/files